app/resources/api/rest/customer/v1/cdr_resource.rb

```diff
@@ -0,0 +1,56 @@
+class Api::Rest::Customer::V1::CdrResource < BaseResource
+  model_name 'Cdr::Cdr'
+
+  key_type :uuid
+  primary_key :uuid
+
+  attributes :time_start,
+             :time_connect,
+             :time_end,
+             :duration,
+             :success, # TODO: or it should be "status"?
```

Comment:
In Admin-panel this field replaced with `status` (text). But in API i think it should be closer to real data. @dmitry-sinina what do you say?

Reply:
I think "success" is good for api